clang/lib/Tooling/DependencyScanning/DependencyScanningWorker.cpp

```diff
@@ -262,6 +279,8 @@ class DependencyScanningAction : public tooling::ToolAction {
   llvm::IntrusiveRefCntPtr<DependencyScanningWorkerFilesystem> DepFS;
   ExcludedPreprocessorDirectiveSkipMapping *PPSkipMappings;
   ScanningOutputFormat Format;
+  const char *ModuleName;
+  const llvm::MemoryBuffer *FakeMemBuffer;
```

Comment:
I think saw Duncan refactoring some instances of `llvm::MemoryBuffer *` to `llvm::MemoryBufferRef` lately. Would it make sense to use that instead?